container_files/assets/www/doc/builtin/procedures/EMProcedure.md

```diff
@@ -0,0 +1,40 @@
+# EM Training Procedure
+
+This procedure trains an Estimation-Maximisation (EM) clustering algorithm with a Gaussian Mixture Model and stores the result mixture model (i.e. cluster centroids and covariance matrix) into an output dataset, as well as storing the cluster labels for its input dataset into a separate output dataset.
+
+## Configuration
+
+![](%%config procedure em.train)
+
+## Training
+
+The EM procedure is a clustering algorithm. It is used to take a set of points, each of which is
+characterized by its coordinates in an embedding space, and group them such
+that each one belongs to one cluster.  The clusters are described by a single
+point that is the cluster centroid, as well as a N by N covariance matrix. 
```

Comment:
It would be great to take an image, even linked to from Wikipedia, that shows how the covariance matrix describes the size and shape of the clusters.  That would greatly help people to understand how to use it.